pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -474,6 +476,14 @@ func (r *ReconcileComplianceSuite) reconcileRemediations(suite *compv1alpha1.Com
 		return reconcile.Result{}, err
 	}
 
+	// We only post-process when everything is done.
+	// This is to prevent unabled to unpause the MachineConfigPool
+	// when there are stucked scans.
+	if suite.Status.Phase != compv1alpha1.PhaseDone {
+		logger.Info("Waiting until all scans are in Done phase before post-processing remediations")
+		return reconcile.Result{}, nil
+	}
+
```

Comment:
Question: This could in theory be a separate patch since this piece of code is doing something else than the rest of the PR, right?

Reply:
Yes, they could be separate patch